MediaBrowser.Model/Dto/UserDataDto.cs

```diff
@@ -0,0 +1,40 @@
+namespace MediaBrowser.Model.Dto
+{
+    /// <summary>
+    /// Class UserDataDto extends UserItemDataDto to allow nullable members.
+    /// This change allow us to implement the new /Users/{UserId}/Items/{ItemId}/UserData endpoint.
+    /// This object allows the requestor to update all or specific user data fields without altering the non-nullable members state.
+    /// </summary>
+    public class UserDataDto : UserItemDataDto
```

Comment:
This name is not descriptive enough. I'd suggest renaming it to "UpdateUserItemDataDto" which is similar to other models we already have like UpdateLibraryOptionsDto.

Reply:
Alright i have renamed them.